### PR TITLE
PTNFLY-1647 - Horizontal Navigation - Make it sticky

### PR DESCRIPTION
**Pull request by junezhang**
https://patternfly.atlassian.net/browse/PTNFLY-1647
Horizontal Navigation - Make it sticky

Change the layout to layout-fixed
Add .navbar-pf to .layout-pf-fixed

### CODE REVIEW

File: less/layouts.less [hunk 1]
```diff
@@ -16,6 +16,13 @@
     body {
       padding-top: @navbar-pf-height; // make space for the navbar
     }
+    .navbar-pf{
+      left: 0;
```
Comment (jeff-phillips-18) [inline less/layouts.less:20]: nit... please add a space before the {

**Comment by jeff-phillips-18** [comment]
LGTM

File: less/layouts.less (hunk 1 above)
Comment (junezhang) [inline less/layouts.less:20]: yes, indeed. Update it.

**Comment by dlabrecq** [comment]
LGTM

**Comment by cardosogabriel** [comment]
@junezhang , @andresgalante asked me to review your PR. The way you did it works, but the CSS you wrote is already supported by bootstrap. You just need to add the class navbar-fixed-top to the component. https://getbootstrap.com/components/#navbar-fixed-top
Can you please update the code and send a new PR? Also, please test in IE 9+.
Many thanks!

**Comment by junezhang** [comment]
@cardosogabriel Thanks for review it.
Yeah, I think it is a long story about the stick nav. Please refrence this link: https://github.com/patternfly/patternfly/pull/397
At last, we decided to keep consistent with the vertical navigation. So adopt this solution.

btw, how do you test IE on Mac, using Browserstack(https://www.browserstack.com/)?

**Comment by cardosogabriel** [comment]
Hi @junezhang. Perhaps I missed the point, but why u guys decided to go for the CSS instead of applying the class to every bar that needs to be fixed?

To test on IE you can use Browserstack, it's awesome. Thanks

**Comment by andresgalante** [comment]
@junezhang @jeff-phillips-18 @dlabrecq  Can you please follow up @cardosogabriel question? Why not use a default bootstrap class and repeat the solution?
By using the bs class we are allowing people to opt for making it sticky or not if they are not using the notification drawer. Thanks!

**Comment by dlabrecq** [comment]
If I recall correctly, the vertical navigation already had its own solution. Both achieved the same result, but we want to use the same approach for the sake of consistency. And changing vertical nav to use the Bootstrap selector now would likely break existing apps?

**Pull request merged**